src/faaslet/Faaslet.cpp

```diff
@@ -8,7 +8,12 @@
 #include <util/timing.h>
 #include <module_cache/WasmModuleCache.h>
 
+#include <wamr/WAMRWasmModule.h>
+#include <wavm/WAVMWasmModule.h>
+
 #if(FAASM_SGX == 1)
+#include <sgx/SGXWAMRWasmModule.h>
```

Comment:
So far this is the minimal dependence I can get on `FAASM_SGX` in this file, but hopefully I can reduce it further in future.